src/main/java/com/icthh/xm/tmf/ms/communication/messaging/MessagingHandler.java

```diff
@@ -39,15 +41,16 @@ private void sendMessage(MessageResponse messageResponse, String topic) {
 
     public void receiveMessage(CommunicationMessage message) {
         Messaging messaging = applicationProperties.getMessaging();
-        List<String> phoneNumbers = new ArrayList<>(from(message).getPhoneNumbers());
+        CommunicationMessageWrapper wrapper = from(message);
+        List<String> phoneNumbers = new ArrayList<>(wrapper.getPhoneNumbers());
         for (String phoneNumber : phoneNumbers) {
             try {
                 RuleResponse ruleResponse = businessRuleValidator.validate(message);
                 if (!ruleResponse.isSuccess()) {
                     failMessage(message, ruleResponse.getResponseCode(), ERROR_BUSINESS_RULE_VALIDATION);
                     return;
                 }
-                String messageId = smppService.send(phoneNumber, message.getContent(), message.getSender().getId());
+                String messageId = smppService.send(phoneNumber, message.getContent(), message.getSender().getId(), wrapper.getDeliveryReport());
```

Comment:
Are you use checkstyle - line length more than 120 symbols.

Reply:
yes, but for some reason code format has not fix the line. Will change it.